lessons/Data-Science/variability/langs/en-us/index.adoc

```diff
@@ -0,0 +1,203 @@
+= Variability
+
+@description{Students define variability multiple ways, and then describe different levels of variability on dot plots.}
+
+@ifproglang{pyret}{
+@lesson-prereqs{contracts-tables-visualizations, ds-intro, dot-plots}
+}
+
+@ifproglang{codap}{
+@lesson-prereqs{codap-dot-plots-bar-charts}
+}
+
+@keywords{variability}
+
+@add-to-lang{histogram}
+
+[@lesson-intro-table]
+|===
+| Lesson Goals
+| Students will be able to...
+
+@objectives
+
+
+| Student-facing Lesson Goals
+|
+
+Let's think about variability of categorical and quantitative data.
+
+| Materials
+|[.materials-links]
+
+@material-links
+
+
+
+|===
+
+
+
+== Variability Two Ways
+
+@objective{variability-define}
+
+=== Overview
+
+Students define variability two ways, and then apply that understanding to describe the variability of categorical and quantitative data.
+
+=== Launch
+
+When we learned about @lesson-link{dot-plots}, we acquired some useful vocabulary that we can use to describe a dataset. We can identify outliers, clusters, peaks, and gaps to characterize a dataset's distribution. We can also estimate what's typical--or what is expected in the data. A third way to describe a dataset is discussing its @vocab{variability}. *There are lots of different ways to think about variability!*
+
+If you have already learned about different _question types_ (perhaps during our lesson on @lesson-link{data-cycle}), you should be familiar with the concept of variability: a @vocab{statistical question} is a question that anticipates @vocab{variability}.
+
+@QandA{
+
+@Q{Which question anticipates variability: _How many minutes are in an hour?_ or _How many minutes does it take to drive to school?_ Explain your response.}
+
+@A{Question A anticipates variability, but Question B does not. The answer Question A will always be 60. The answer to the Question B, however, will vary based on who you ask, road conditions, traffic, etc.}
+}
+
+@strategy{There are Many Ways to Think about Variability!}{
+
+Research indicates that students often have an oversimplified and underdeveloped view of variability (@citation{cooper-2018}; @citation{cooper-shore-2008}).
+
+In this lesson, we intentionally begin our conversation by developing intuitive ideas about variability, for instance:
+
+- Variability requires us to consider the data as an entity, rather than as individual points.
+- We can try to understand why things vary and try to identify reasons for variability.
+- Some things vary a little, and some vary a lot.
+- We see variability in both quantitative and categorical datasets.
+
+This last recommendation is an important one: research indicates that "unlikeability" of a categorical dataset is a more natural concept than "variation about the mean" (@citation{kade-perry-2007}), therefore can act as a natural starting point.
```

Comment:
You clearly go this from the literature but I object to using the word "unlikeability" here. It sounds like we're talking about how much we dislike the data.

The word unlikeable means "hard to like".
Unlikeability itself neither exists in Webster's nor am I seeing it come up anywhere else.

Far more important than using the researchers' strange word choice is making the idea they are trying to communicate come across. And it seems the word that communicates what they intend to communicate that actually exists in the English language is "unlike".

I will replace it with the following rewrite.
> This last recommendation is an important one: research indicates that it is more natural to understand how like or unlike categorical data is than it is to understand variation about the mean (@citation{kade-perry-2007}), therefore reasoning about variability in categorical datasets can act as a natural starting point.